src/coreclr/scripts/superpmi.py

```diff
@@ -402,15 +408,15 @@ def write_file_to_log(filepath, log_level=logging.DEBUG):
     if not os.path.exists(filepath):
         return
 
-    logging.log(log_level, "============== Contents of %s", filepath)
+    logging.log(log_level, "%s============== Contents of %s", get_time_prefix(), filepath)
```

Comment:
Don't think printing time here is of any use. The real time prefix needs to happen at the point where the contents in the `filepath` were created.

Reply:
It's true it might not be what we might want to see, but it does provide at least some information.